spec.bs

```diff
@@ -320,26 +421,243 @@ maps to an internal [=fenced frame config=] [=struct=].
 
 Issue: Fix the "indistinguishable" IDL bug with the unions above.</span>
 
+<div algorithm>
+  In order to <dfn export>lift a [=urn uuid=] and [=fenced frame config=] into a
+  {{FencedFrameConfig}}</dfn>, run these steps:
+
+  1. <span class=XXX>TODO: Fill this in</span>
+</div>
+
 Each {{FencedFrameConfig}} has a <dfn for=fencedframeconfig>url</dfn>, which is a string, initially
-null.
+null. TODO: flesh out {{FencedFrameConfig}}
+
+A key feature of the <{fencedframe}> element is that web platform APIs can configure the behavior
+of the frame in a way that limits the ability of other execution contexts to modify or inspect this
+configuration, for security or privacy reasons. For example, the
+<a href=https://wicg.github.io/turtledove/>FLEDGE</a> API performs on-device ad auctions over
+cross-site data, and it is important that the ad that wins the auction can be loaded into a frame,
+without the API caller knowing *which ad* won the auction or being able to manipulate the
+environment in which the ad loads.
+
+We achieve this using the concept of a "fenced frame config". A fenced frame config is a collection
+of fields that can be loaded into <{fencedframe}> elements and that specifies the resulting
+environments. Fenced frame configs can only be constructed by web platform APIs, not initialized
+or modified arbitrarily. Their fields also contain "visibilities" for different entities, which
+dictate whether the field should be "redacted" when inspected by particular execution contexts.
+Each time a fenced frame config is loaded into a <{fencedframe}> element, it is instantiated as a
+new "fenced frame config instance", which governs that particular context.
```

Comment:
These two paragraphs are very informal and don't do a lot of linking or describe normative requirements. I'd recommend one of two things:
 1. Make this into a sort of "Introduction" subsection with the `*This section is non-normative*` disclaimer, and then make the following bits with the "preliminary types" into its own section called "Supporting concepts"
 2. Formalize these two paragraphs a bit and add a bunch of linking, and turn the "For example" stuff into its own example or note block.

I slightly prefer (1), but will leave it to you.

Reply:
Made it a non-normative intro.